packages/react-native-client/src/hooks/useCamera.ts

```diff
@@ -13,13 +13,14 @@ import RNFishjamClientModule from '../RNFishjamClientModule';
 import { ReceivableEvents, useFishjamEvent } from './useFishjamEvent';
 
 type SimulcastConfigUpdateEvent = SimulcastConfig;
-export type CaptureDeviceId = Brand<string, 'CaptureDeviceId'>;
+export type CameraId = Brand<string, 'CameraId'>;
 
-export type CaptureDevice = {
-  id: CaptureDeviceId;
+export type CameraFacingDirection = 'front' | 'back' | 'other';
```

Comment:
**suggestion:** Maybe `unspecified` will be more descriptive in this case and it follows iOS naming convention.